README.md

```diff
@@ -116,33 +116,39 @@ establish "reproducible interactive sessions" with the help of that script.
 
 Singularity image files have `.sing` extension.  Since we are providing
 a custom filename to store the file at, we cannot guess the format of
-the container (e.g., either it is 
+the container (e.g., either it is
 [.sif](https://www.sylabs.io/2018/03/sif-containing-your-containers/)),
 so we just use uniform `.sing` extension.
 
 # A typical YODA workflow
 
-Before proceeding our typical workflow, let's re-express [YODA principles] as a possible workflow:
+Lets summarize YODA principles as a possible workflow:
 
-- create dataset which would contain results,
-- install/add to that dataset everything needed (code, other datasets, containers)
-  to perform the analysis
+- create a new dataset which would contain results and everything needed
+    to obtain them
+- install/add subdatasets(code, other datasets, containers)
 - perform the analysis using **only** materials available within the reach of this dataset.
 
+
+Let's assume that our goal is to do Quality Control of an MRI dataset
+(which is available as DataLad dataset ds000003. We will create a new
+dataset with the output of the QC results(as analyzed by mriqc
+BIDS-App). mriqc is provided by the ReproNim/containers dataset of
+containers. Below, we execute a simple analysis workflow which 
+adheres to YODA principles and we **end up with a dataset that contains
+all componentes necessary a history of how it was achieved.**
```

Comment:
is spellchecker enabled?
```suggestion
all components necessary a history of how it was achieved.**
```

Reply:
I have now added codespell workflow to this repo.  Please rebase your PR on top of master and repush -- let's see if green ;-)